docs/modules/ROOT/pages/references/quality-requirements/functional/minimal-invoiced-amounts.adoc

```diff
@@ -0,0 +1,26 @@
+= Minimal invoiced amounts
+
+Source::
+Invoicing adapter
+
+Stimulus::
+Calculate invoice
+
+Environment::
+{global}
+
+Artifact::
+Enterprice resouce planning or accounting system where the invoices get created.
+
+Response::
+Line items on the generated invoices for the billed resources.
+
+Response measure::
+A minimal amount gets billed even if the effective usage is lower.
+
+Memory::: 125MB
+Disk storage::: 1GB
+
+Rationale::
+Allocating compute and storage resources on a Kubernetes cluster binds some resources on the clusters control plane.
+The minimal billed amount accounts for that overhead.
```

Comment:
It's not quite clear how this "line" for "billed resources" is summarized.
Therefore my question: Is this minimum per pod, or per namespace, per zone?
What does this line contain, can you provide an example?

Considering a namespace with 5 pods, each consuming 10 MBs. Is each Pod billed with 125 MB? Or is the namespace billed with 125 MB (covering all 5 pods = 50 MBs) since that is the "billed line"?